packages/forms/backend/sirius-components-forms/src/main/java/org/eclipse/sirius/components/forms/TreeNode.java

```diff
@@ -36,8 +38,16 @@ public final class TreeNode {
 
     private List<String> iconURL;
 
+    private List<List<String>> iconEndURL;
```

Comment:
I would prefer `endIconsURL` instead of `iconEndURL`. Could you rename everywhere please?